### PR TITLE
Default states fixes

### PR DESCRIPTION
**Pull request by fdansv**
* Histogram bars are not passed in the state if they haven't been changed
* Map state will not be reported if the position hasn't changed from 1% of its default position.

### CODE REVIEW

**Comment by fdansv** [comment]
@alonsogarciapablo mind taking a quick look? ;)

File: src/api/dashboard.js [hunk 1]
```diff
@@ -45,11 +45,17 @@ Dashboard.prototype = {
     var initialState = this._dashboard.dashboardView.getInitialMapState();
     var currentCenter = this._dashboard.vis.map.get('center');
     var currentZoom = this._dashboard.vis.map.getZoom();
-    if (_.isEqual(currentCenter, initialState.center) && _.isEqual(currentZoom, initialState.zoom)) return {};
-    return {
-      center: currentCenter,
-      zoom: currentZoom
-    };
+    var upperLat = initialState.center[0] * 0.01;
```
Comment (alonsogarciapablo) [inline src/api/dashboard.js]: I'd be easier for me to understand what this change is about if there was a test for it...

Comment (alonsogarciapablo) [inline src/api/dashboard.js]: What is `0.01`? I read the PR description and I know that is a 1% of something... but my (or even your) future self might not remember what this is... Could we declare a "constant" and give a name?

Comment (alonsogarciapablo) [inline src/api/dashboard.js]: Also, we could make this a bit clearer/easier to understand by extracting a method:

```
if (this._hasStateChanged(...)) {
  return {
    center: currentCenter,
    zoom: currentZoom
  };
} else {
  return {};
}
```

**Comment by alonsogarciapablo** [comment]
It makes sense. I left some comments/suggestions to make it a bit clearer and I wish there were some tests (specially for the part that implements `Map state will not be reported if the position hasn't changed from 1% of its default position`).

**Pull request merged**